pkg/controller.v1/common/job_test.go

```diff
@@ -219,6 +219,40 @@ func TestPastActiveDeadline(T *testing.T) {
 	}
 }
 
+func TestManagedBy(T *testing.T) {
+	cases := map[string]struct {
+		managedBy string
```

Comment:
```suggestion
		managedBy *string
```

So that we can avoid the custom insertion loginc in the following, could we make this pointer?

```
if tc.managedBy == "" {
	runPolicy.ManagedBy = nil
}
```

Reply:
+1